packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -2696,8 +2697,26 @@ describe("Runtime", () => {
 			});
 
 			it("ignores in-flight signals on disconnect/reconnect", () => {
+				// eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+				(containerRuntime as any).channelCollection = {
+					setConnectionState: (_connected: boolean, _clientId?: string) => {},
+					// Pass data store op right back to ContainerRuntime
+					reSubmit: (type: string, envelope: any, localOpMetadata: unknown) => {
+						submitDataStoreOp(
+							containerRuntime,
+							envelope.address,
+							envelope.contents,
+							localOpMetadata,
+						);
+					},
+				} as ChannelCollection;
```

Comment:
You should comment this. It is unusual to change the details of the thing being tested.